scripts/test-node.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
What happens if you exit with a non zero exit code here? Would that not make the failed test more visible than simply having to view the log?

It would be nice if it actually told you that the image produced is not ready for consumption.